taskcluster/ci/config.yml

```diff
@@ -57,6 +57,13 @@ taskgraph:
             default-repository: git@github.com:mozilla-extensions/cpmanager
             default-ref: master
             type: git
+        httpsupgradestudyv2:
```

Comment:
This needs to match the `repo-prefix`. In this case you set it to `pioneer-v2-example`... I'm not sure it accepts dashes though.